.github/workflows/coding-statndard-check.yml

```diff
@@ -0,0 +1,70 @@
+name: PHP Coding Standard Check
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  php-cs-fixer:
+    name: PHP-CS-Fixer
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@master
+
+      - name: PHP-CS-Fixer
+        uses: docker://oskarstark/php-cs-fixer-ga:3.4.0
+        with:
+          args: --format=txt --diff --dry-run --using-cache=no --verbose .
+
+  phpstan:
+    name: PHPStan
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        php-version:
+          - "7.1"
```

Comment:
in the PR you said we support from 5.6 to 8.1 so should we perform a test on 5.6 as well?

Reply:
PHPStan do not support PHP 5.6 and 7.0. That's why those versions are not included.

We ran integration test locally from PHP 5.6 to 8.1 and it passes. We removed tests from Github action because we want it to run in the SonarCloud.